userchroot.c

```diff
@@ -208,6 +226,98 @@ static void portable_clearenv() {
 #endif
 }
 
+struct epilogue_data {
+    uid_t target_user;
+    char** argv;
+    char** envp;
+};
+
+void epilogue(struct epilogue_data* d) {
```

Comment:
I think I'd rather just pass the values individually instead of creating that struct...